src/osgEarthDrivers/gdal/ReaderWriterGDAL.cpp

```diff
@@ -651,6 +661,16 @@ build_vrt(std::vector<std::string> &files, ResolutionStrategy resolutionStrategy
 
         for(j=0;j<nBands;j++)
```

Comment:
Is it really per band that you want the info? So the result will be for example 100 (bands) * 10 (tiles) for example.
Also it think that this algorithm wil write 100 times the same value which is not efficient/usefull.
I think that that we want the list of received files extent.